addons/udes_stock/models/stock_move.py

```diff
@@ -361,6 +361,24 @@ def group_by_key(self):
 
         return self.groupby(lambda ml: ml.u_grouping_key)
 
+    def _prepare_extra_values_for_group(self, pickings, moves):
```

Comment:
You'll actually get the extra values only if the collections below will have a single element. Is this safe enough for the caller? Also, the method name doesn't say anything about what's going to happen... You gotta read the code.

Reply:
Thanks. Will add docstring, to explain.